hotshot-state-prover/src/service.rs

```diff
@@ -556,7 +560,11 @@ mod test {
 
         let (genesis, _qc_keys, state_keys, st) = init_ledger_for_test();
 
-        let anvil = Anvil::new().spawn();
+        let anvil = {
+            let port = portpicker::pick_unused_port().expect("No ports free");
```

Comment:
Ethers should already handle this https://docs.rs/ethers-core/2.0.13/src/ethers_core/utils/anvil.rs.html#222

Reply:
removed all those in: https://github.com/EspressoSystems/espresso-sequencer/commit/447115188afc4855b2a1f6e56535af1675dc9356